packages/arb-token-bridge-ui/src/hooks/useTransactionHistory.ts

```diff
@@ -670,7 +670,7 @@ export const useTransactionHistory = (
       }
 
       // ETH deposit
-      if (tx.asset === AssetType.ETH) {
+      if (tx.assetType === AssetType.ETH) {
```

Comment:
Hmm wouldn't a better check be using the values returned by `useNativeCurrency` here?

Reply:
`assetType` won't change and should always be 'ETH'. @spsjvc was also in agreement on this